airbyte-integrations/connectors/destination-bigquery/src/main/kotlin/io/airbyte/integrations/destination/bigquery/write/BigqueryRawTableOperations.kt

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination.bigquery.write
+
+import com.google.cloud.bigquery.BigQuery
+import com.google.cloud.bigquery.QueryJobConfiguration
+import com.google.cloud.bigquery.TableId
+import com.google.cloud.bigquery.TableResult
+import io.airbyte.cdk.load.message.Meta
+import io.airbyte.cdk.load.orchestration.TableName
+import io.airbyte.cdk.load.orchestration.legacy_typing_deduping.TypingDedupingRawTableOperations
+import io.airbyte.integrations.destination.bigquery.BigQueryUtils
+import io.airbyte.integrations.destination.bigquery.formatter.BigQueryRecordFormatter
+import io.github.oshai.kotlinlogging.KotlinLogging
+
+private val logger = KotlinLogging.logger {}
+
```

Comment:
Taken from BigqueryStorageOperation.

one note: we used to have separate impls for standard inserts and GCS. AFAICT the only difference was that in GCS mode:
* we did a gcs.createBucket call (which we want to stop doing)
* we did a gcs.deleteObjects call at the end of the sync (which is now handled in the bigquery bulk loader)

so we can combine them now.